### PR TITLE
Added examples at Basic Commands chapter Seeking section

### PR DESCRIPTION
**Pull request by FreeArtMan**

### CODE REVIEW

File: basic_commands/seeking.md [hunk 1]
```diff
@@ -37,3 +37,67 @@ If you want to inspect the result of a math expression, you can evaluate it usin
 
 
 In the visual mode you can press `u` (undo) or `U` (redo) inside the seek history to return back to previous or forward to the next location.
+
+## Open file
+
+As test file lets use some simple hello_world.c compiled in Linux ELF format.
+After we compiled it lets open it with radare2
+
+    r2 ./hello_world
```
Comment (radare) [inline basic_commands/seeking.md]: No need to prefix with ./

File: basic_commands/seeking.md [hunk 2]
```diff
@@ -37,3 +37,67 @@ If you want to inspect the result of a math expression, you can evaluate it usin
...
+
+    [0x00400410]>
+
+Now we are ready to go deeper. 
+
+## Seeking at any position
+
+Seek to address 0x0
+
+    [0x00400410]> s 0x0
```
Comment (radare) [inline basic_commands/seeking.md]: Numeric input can be in any base. No need to overcomplicate with 0x

File: basic_commands/seeking.md [hunk 3]
```diff
@@ -37,3 +37,67 @@ If you want to inspect the result of a math expression, you can evaluate it usin
...
+    [0x00000000]>
+
+Print current address 
+
+    [0x00000000]> s
+    0x0
+    [0x00000000]>
+
```
Comment (radare) [inline basic_commands/seeking.md]: Can be also done with ?v $$

File: basic_commands/seeking.md [hunk 4]
```diff
@@ -37,3 +37,67 @@ If you want to inspect the result of a math expression, you can evaluate it usin
...
+Seek N positions forward
+
+    [0x00000000]> s+ 128
```
Comment (radare) [inline basic_commands/seeking.md]: That space is optional

File: basic_commands/seeking.md [hunk 5]
```diff
@@ -37,3 +37,67 @@ If you want to inspect the result of a math expression, you can evaluate it usin
...
+    [0x00000080]>
+
+Undo/Redo seek. Lets see with positions we where seeking
+
+    [0x00400410] [0x00000000] [0x00000080]
```
Comment (radare) [inline basic_commands/seeking.md]: The seek history should be listed with the s! Command or the us .. I gues we need to fill an issue for that but i remember there was a way to do that but incant find it from the phone now.

File: basic_commands/seeking.md [hunk 6]
```diff
@@ -37,3 +37,67 @@ If you want to inspect the result of a math expression, you can evaluate it usin
...
+
+Lets undo last two seeks to beginning
+
+    [0x00000080]> s-
+    [0x00000000]> s-
+    [0x00400410]>
+
+Now we are at same position as at beginning _0x00400410_.
+
+Lets search in hellow_world ELF file 'Hello'. After search our position will set
```
Comment (radare) [inline basic_commands/seeking.md]: Hello vs hellow

File: basic_commands/seeking.md [hunk 7]
```diff
@@ -37,3 +37,67 @@ If you want to inspect the result of a math expression, you can evaluate it usin
...
+    Searching 5 bytes from 0x00400411 to 0x00600928: 48 65 6c 6c 6f 
+    Searching 5 bytes in [0x400411-0x600928]
+    hits: 1  hit0_0 .. hit0_0
+    0x004005b4 hit0_0 "Hello"
+    [0x004005b4]>s-
+    [0x00400410]>
+
+There is history command to see all our changes
+
+    [0x00400410]> s*
```
Comment (radare) [inline basic_commands/seeking.md]: Oh right. It was s* :D not s!

File: basic_commands/seeking.md (hunk 3 above)
Comment (FreeArtMan) [inline basic_commands/seeking.md]: Should alternatives to some commands also provided?

File: basic_commands/seeking.md (hunk 2 above)
Comment (FreeArtMan) [inline basic_commands/seeking.md]: All params could be in any base? Could be it described in some previous chapter as basics of using radare2?

Comment (Maijin) [inline basic_commands/seeking.md]: Yes could be added somewhere feel free to add it :)

File: basic_commands/seeking.md (hunk 3 above)
Comment (radare) [inline basic_commands/seeking.md]: Yes, if necessary, just to show some use examples, no need to be fully detailed

File: basic_commands/seeking.md (hunk 2 above)
Comment (radare) [inline basic_commands/seeking.md]: Please update the pr

**Comment by radare** [comment]
Thanks!

**Pull request merged**